plugins/woocommerce/client/blocks/assets/js/atomic/utils/render-parent-block.tsx

```diff
@@ -216,54 +216,58 @@ const renderInnerBlocks = ( {
 			? blockWrapper
 			: Fragment;
 
-		return (
-			<Suspense
-				key={ `${ block }_${ depth }_${ index }_suspense` }
-				fallback={ <div className="wc-block-placeholder" /> }
+		const isCheckoutBlock = block === 'woocommerce/checkout';
```

Comment:
Cart will also be updated. The only block using the suspense after this might be the Mini Carts contents